packages/clerk-js/package.json

```
@@ -75,6 +75,7 @@
     "@swc/helpers": "^0.5.17",
     "@zxcvbn-ts/core": "3.0.4",
     "@zxcvbn-ts/language-common": "3.0.4",
+    "alien-signals": "^2.0.5",
```

Lets be strict about this as it will land in clerk-js even during the beta I think
```suggestion
    "alien-signals": "2.0.5",
```